firmware/gibibot_firmware/Debug/Core/Src/main.d

```diff
@@ -0,0 +1,81 @@
+Core/Src/main.o: ../Core/Src/main.c ../Core/Inc/main.h \

```

Comment:
Delete `Debug` folder and add to `.gitignore`. 
`Debug` and `Release` are build folders and should not be in version control.